### ISSUE TITLE
Cannot achieve Convergence with git-version

### ISSUE THREAD

**Issue by Mixpap** [editeodoro/Bbarolo#15]
After compiling the latest git version of Bbarolo the code cannot achieve convergence where with the same configuration file/data and the stable version it can. 

Tried with the DELTAVROT option on and off. Both examples tried to fit VROT VRAD INC VDISP PA

Michalis

**Comment by editeodoro** [comment]
Hi Michalis, 
thanks for reporting this, I had messed up with some array indexes. It should be fixed now. Let me know if you see any additional issue. 

Note: I had to add some degree of randomness in the re-initialization of the simplex when the algorithm samples regions outside the boundaries. This causes the algorithm to converge to slightly different minima at each run when the boundaries are not large enough to sample properly the parameter space. The minimum however looks always consistent within the errors, thus it should not be a big deal. 

Enrico
